src/Uno.UI/UI/Xaml/Style/Generic/GenericStyles.Android.cs

```diff
@@ -33,18 +32,9 @@ public static partial class GenericStyles
 			InitPage();
 			InitWebView();
 			InitDatePicker();
-			InitFlipViewItem();
 			InitToggleSwitch();
 			InitFlyoutPresenter();
 
-#if !IS_UNO
```

Comment:
why so many deletes?

Reply:
@jeremiethibeault These were the changes you had asked me to do in my initial Pull Request that was part of VSTS. I assume these changes are still useful?

Reply:
@MatFillion This is all dead code referring to Umbrella controls (apart from InitContentControl - I don't know what's the deal with that)

Reply:
Mmmm I must have been misunderstood in the past! No idea what we are talking about. :)